src/Controllers/team.controller.ts

```diff
@@ -51,8 +66,8 @@ export const getTeam = async (req: Request<{ pid: string }>, res: Response) => {
   }
 
   const team = await prisma.team.findUnique({
-    where: { pid },
-    select: basicTeam,
+    where: { pid: pid },
```

Comment:
This changes nothing (but is not a problem either).

_See_: https://attacomsian.com/blog/javascript-object-property-shorthand